### ISSUE TITLE
[Suggestion]: Some open sky between all the gloom

### ISSUE THREAD

**Issue by shockr** [nonamecrackers2/simple-clouds#42]
### Guidelines

- [X] By checking this box, I am saying that I have read the guidelines and believe to be following them to the best of my ability

### Description of feature request

Not really much of a request or suggestion, but it would be nice to have some clear sky in between. Perhaps a variable for the space between cloud  formations.

Very pretty, but it just seems to be constantly dark and rainy.. and I'm from the UK!

**Comment by shockr** [comment]
I guess, a good suggestion might be a way to add weights to the different formations so a big storm might only roll in once in a while?

**Comment by LunaticWyrm467** [comment]
Yeah, I would really prioritize showers and thundershowers over more complex storms and fronts.
This prioritization should be fine to do with a cumulonimbus cloud region as cumulonimbus clouds would work fine as being naturally localized phenomena.

However, maybe a cloud region could be added that's somewhere between a cumulus and cumulonimbus cloud region (E.g. cumulus congestus), which brings scattered rain showers as well (since nimbostratus/stratus clouds don't make sense to localize as they are typically associated with longer storms).

**Comment by ebo2022** [comment]
> Yeah, I would really prioritize showers and thundershowers over more complex storms and fronts. This prioritization should be fine to do with a cumulonimbis cloud region as cumulonimbus clouds would work fine as being naturally localized phenomena.
> 
> However, maybe a cloud region could be added that's somewhere between a cumulus and cumulonimbus cloud region (E.g. cumulus congestus), which brings scattered rain showers as well (since nimbostratus/stratus clouds don't make sense to localize as they are typically associated with longer storms).

I agree about prioritizing the pop-up/small-scale storms and adding an intermediate type between cumulus and cumulonimbus. 

My only note here is I don't think individual storms should contain any potential "severe" weather (with a [few exceptions](https://en.wikipedia.org/wiki/Pulse_storm)). That should be relegated to organized systems/fronts, which a player should - in my opinion - be able to predict within reasonable extent if they know what they're looking for.

I still think fronts have their place (hence why I wrote a wordwall explaining why) but they shouldn't be overly common and certainly not the only form of stormy weather.

**Comment by shockr** [comment]
A temporary resolution to this would be to create some 'ClearSky' custom clouds and potentially even some duplicates of cloud formations that were favourable over storms. 

Now if I could just figure out how to get the mod to load my saved json files from the editor kindly provided.

**Comment by swedneck** [comment]
Putting it here for reference: You can add custom clouds to the jar file if nothing else, just open it in whatever archive program and drop the json files into data/simpleclouds/cloud types and then the cloud types should show up when you load a world.

**Comment by nonamecrackers2** [comment]
Please don't modify the jar, instead you can do this with a datapack

**Comment by shockr** [comment]
So, I can put my new json files in a datapack on my dedicated server? (<world>/datapacks/packname/data/simpleclouds/cloud_types/)

Or do I need to somehow crowbar this into the modpack people use on my server?

**Comment by nonamecrackers2** [comment]
You can just put it in a datapack on your server and it should be synced to all the clients

**Comment by nonamecrackers2** [comment]
Will be added in the next release
